known-urls.txt

```diff
@@ -0,0 +1,19 @@
+
+> zoo-modeling-app@0.0.0 url-checker
+> ./scripts/url-checker.sh
+
+URL  STATUS
+000  https://${BASE_URL}
+301  https://discord.gg/JQEpHR7Nt2
+404  https://github.com/KittyCAD/engine/issues/3528
+302  https://github.com/KittyCAD/modeling-app/issues/new/choose
+302  https://github.com/KittyCAD/modeling-app/issues/new?template=bug_report.yml
+302  https://github.com/KittyCAD/modeling-app/issues/new?title=${title}&body=${body}
+404  https://github.com/KittyCAD/modeling-app/releases/tag/v${version}
+521  https://placekitten.com/200/200
+302  https://reactrouter.com/en/6.16.0/routers/picking-a-router#using-v64-data-apis
+302  https://stackoverflow.com/a/57390160/22753272
+302  https://stackoverflow.com/a/58436959/22753272
+301  https://text-to-cad.zoo.dev/dashboard
+307  https://zoo.dev/
+308  https://zoo.dev/docs/api/ml/generate-a-cad-model-from-text
```

Comment:
I'm not sure we need this as another top-level file. Perhaps something like `scripts/output/` could work? Open to suggestions.

Reply:
I can move this, again I followed jess's pattern of `known-circular.txt`. This is the one file we want to use in the CI CD testing. 

I'd wanna move all of these IMO so they follow the same pattern. This is the second script that we run and have a whitelist that needs to be updated otherwise a future test would fail.